platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/PlatformData.java

```diff
@@ -504,15 +559,7 @@ public boolean equals(final Object o) {
      */
     @Override
     public int hashCode() {
-        return new HashCodeBuilder(17, 37)
-                .append(round)
-                .append(numEventsCons)
-                .append(hashEventsCons)
-                .append(events)
-                .append(consensusTimestamp)
-                .append(minGenInfo)
-                .append(epochHash)
-                .toHashCode();
+        return NonCryptographicHashing.hash32(round);
     }
 
     /**
```

Comment:
Should `minimumGenerationNonAncient` be added to `toString`?

Reply:
added